### DISCUSSION TITLE
LastException : Help request, error code with version 1.103.315.1020

### DISCUSSION THREAD

**Discussion by heidi137** [TitanNano/TS4ControlAnySim#73] [Q&A] [answered]
![Capture d’écran 2023-12-27 082011](https://github.com/TitanNano/TS4ControlAnySim/assets/14272630/1c9c8c18-a3ad-4e27-9fac-b536df64eb09)

An error appears on the report, with a 30% chance that it's the Control Any Sim that's causing it. Can you run a test to see if everything's okay with the new version of Sims4? VIP has declared it broken, I have it in game, it works, but leaves an error when opening the game, it's recurring. Thanks for your help, I love this mod so much. heidi137 on MTS

**Comment by TitanNano** [comment]
This should not have anything to do with the mod. Perhaps you can post the full exception details and not just the summary to shine more light on where the exception is originating from.

**Comment by heidi137** [comment]
It would be kind of you to help me, the diagnosis is approximate, that's
why I took the liberty of asking you, on VIP, Scarlett noted the Control
any Sim as a broken mod. It works apart from that error message that comes
in game entry, every game entry and every lot download.
Thanks for the help, with L051's library, the 50/50 is complicated, the
game doesn't open anymore sometimes, elements are missing.

Le jeu. 28 déc. 2023 à 00:16, Jovan Gerodetti ***@***.***> a
écrit :

> This should not have anything to do with the mod. Perhaps you can post the
> full exception details and not just the summary to shine more light on
> where the exception is originating from.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TitanNano/TS4ControlAnySim/discussions/73#discussioncomment-7960902>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADM4Q5XCLKKQL5RA2UVPOKLYLST3LAVCNFSM6AAAAABBEBF5UOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TSNRQHEYDE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

"J'aime la force, et de la force que j'aime, une fourmi peut en montrer
autant qu'un éléphant."
[Stendhal]
"Il faut accepter de planter pour que d'autres récoltent ailleurs et plus
tard."
[Bernard Werber]

<?xml version="1.0" ?><root>
<BetterExceptions><BEversion>v3.12</BEversion><WasBlank>False</WasBlank><TuningLoadFinished>True</TuningLoadFinished><BadObjectCC>False</BadObjectCC><Advice>Not available. More info may be in BE Report.</Advice></BetterExceptions><report><version>2</version><sessionid>6f506dcd373385fc658b4db8</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-12-26 23:49:37</createtime><buildsignature>Local.Unknown.Unknown.1.103.315.1020-1.300.000.175.Release</buildsignature><categoryid>trait_tracker.py:1738</categoryid><desyncid>6f506dcd373385fc658b4db8</desyncid><systemconfig/><screenshot/><desyncdata>[BE Interceptor] BE intercepted this error from a separate mod log so that it can scan for the issue. (AttributeError: 'NoneType' object has no attribute 'min_lod_value')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 130, in wrapped&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 126, in finally_wrap&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 157, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 175, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 383, in c_api_client_connect&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\masked\v49\injector.py", line 26, in _wrapped_function&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\masked\v49\zone_spin.py", line 100, in _turbolib_on_late_zone_load&#13;&#10;  File ".\ImmersiveFarting\immersive_farting\__init__.py\__init__.py", line 178, in _wrapped_function&#13;&#10;  File ".\ImmersiveFarting\immersive_farting\__init__.py\__init__.py", line 199, in _immersive_farting_on_late_zone_load&#13;&#10;  File "noseewhatshappening\inject.py", line 17, in _wrapped_function&#13;&#10;  File "noseewhatshappening\inject.py", line 34, in _common_on_late_zone_load&#13;&#10;  File "src/control_any_sim/util/inject.py", line 10, in _inject&#13;&#10;  File "src/control_any_sim/util/game_events.py", line 84, in canys_zone_do_zone_spin_up&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone.py", line 608, in do_zone_spin_up&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\time_service.py", line 117, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling\masked\simulation\unclog\__init__.py", line 25, in _wrapped_function&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling\masked\simulation\unclog\__init__.py", line 53, in _gtu_on_sim_timeline_simulation&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 224, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 385, in _resume&#13;&#10;  File ".\WonderfulWhims_v49\turbolib2\events\sims.py", line 135, in _turbolib_sim_startup_sim_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\sim.py", line 2949, in _startup_sim_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 1337, in on_sim_startup&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 1213, in _add_buffs&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\__init__.py", line 470, in exported_func&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\buff_component.py", line 344, in add_buff&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\buffs\buff.py", line 1215, in on_add&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\buffs\buff.py", line 1263, in apply_all_loot_actions&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\utils\loot.py", line 595, in apply_to_resolver&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\utils\loot_basic_op.py", line 303, in apply_to_resolver&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\utils\loot_ops.py", line 389, in _apply_to_subject_and_target&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\occult\sim_info_with_occult_tracker.py", line 74, in add_trait&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 84, in add_trait&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 654, in _add_trait&#13;&#10;  File "E:\Builds\MCCC_2023_7_1\mc_cmd_center\mc_utils.py", line 202, in _wrapper_function&#13;&#10;  File "E:\Builds\MCCC_2023_7_1\mc_cas\mc_cas_inject.py", line 180, in inject_can_add_trait&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 426, in can_add_trait&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\traits\trait_tracker.py", line 1738, in _has_valid_lod&#13;&#10;AttributeError: 'NoneType' object has no attribute 'min_lod_value'rtim=0&#13;&#10;Client Session Time: 163.63&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,86,0,5599&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1060 6GB nvldumdx.dll 31.0.15.3758, GUID: D7B71E3E-5F43-11CF-7C79-8D120EC2D235&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>6f506dcd373385fc658b4db8</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-12-26 23:50:27</createtime><buildsignature>Local.Unknown.Unknown.1.103.315.1020-1.300.000.175.Release</buildsignature><categoryid>proximity_component.py:220</categoryid><desyncid>6f506dcd373385fc658b4db8</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Wall clock Timeline: Exception running Element (AttributeError: 'NoneType' object has no attribute 'proximity_detection_tests')&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 219, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\alarms.py", line 260, in _run&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\proximity_component.py", line 184, in _initial_update_callback&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\proximity_component.py", line 220, in _pulse&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\proximity_component.py", line 301, in _add_and_remove_buffs_for_simAttributeError: 'NoneType' object has no attribute 'proximity_detection_tests'rtim=0&#13;&#10;Client Session Time: 163.63&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,86,0,5599&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1060 6GB nvldumdx.dll 31.0.15.3758, GUID: D7B71E3E-5F43-11CF-7C79-8D120EC2D235&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>6f506dcd373385fc658b4db8</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-12-26 23:50:28</createtime><buildsignature>Local.Unknown.Unknown.1.103.315.1020-1.300.000.175.Release</buildsignature><categoryid>proximity_component.py:220</categoryid><desyncid>6f506dcd373385fc658b4db8</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Wall clock Timeline: Exception running Element (AttributeError: 'NoneType' object has no attribute 'proximity_detection_tests')&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 219, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\alarms.py", line 260, in _run&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\proximity_component.py", line 184, in _initial_update_callback&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\proximity_component.py", line 220, in _pulse&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\proximity_component.py", line 301, in _add_and_remove_buffs_for_simAttributeError: 'NoneType' object has no attribute 'proximity_detection_tests'rtim=0&#13;&#10;Client Session Time: 163.63&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,86,0,5599&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1060 6GB nvldumdx.dll 31.0.15.3758, GUID: D7B71E3E-5F43-11CF-7C79-8D120EC2D235&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>6f506dcd373385fc658b4db8</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2023-12-26 23:50:28</createtime><buildsignature>Local.Unknown.Unknown.1.103.315.1020-1.300.000.175.Release</buildsignature><categoryid>proximity_component.py:220</categoryid><desyncid>6f506dcd373385fc658b4db8</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Wall clock Timeline: Exception running Element (AttributeError: 'NoneType' object has no attribute 'proximity_detection_tests')&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 219, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\alarms.py", line 260, in _run&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\proximity_component.py", line 184, in _initial_update_callback&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\proximity_component.py", line 220, in _pulse&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\proximity_component.py", line 301, in _add_and_remove_buffs_for_simAttributeError: 'NoneType' object has no attribute 'proximity_detection_tests'rtim=0&#13;&#10;Client Session Time: 163.63&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,86,0,5599&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1060 6GB nvldumdx.dll 31.0.15.3758, GUID: D7B71E3E-5F43-11CF-7C79-8D120EC2D235&#13; &#10;</desyncdata></report>
</root>

**Comment by heidi137** [comment, marked as answer]
Here I am again, Mal22's tracker was missing and a feature I need to
identify, I've found. The BE made me take a lot of things out of my game,
it's unreliable for my part, too bad it's authentic for the list of broken
mods, yours is listed in it, I've already told Scarlett it works, but it's
still noted broken. Thanks for your attention, it's nice to be heard, thank
you.😉

Le jeu. 28 déc. 2023 à 00:16, Jovan Gerodetti ***@***.***> a
écrit :

> This should not have anything to do with the mod. Perhaps you can post the
> full exception details and not just the summary to shine more light on
> where the exception is originating from.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TitanNano/TS4ControlAnySim/discussions/73#discussioncomment-7960902>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADM4Q5XCLKKQL5RA2UVPOKLYLST3LAVCNFSM6AAAAABBEBF5UOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TSNRQHEYDE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

"J'aime la force, et de la force que j'aime, une fourmi peut en montrer
autant qu'un éléphant."
[Stendhal]
"Il faut accepter de planter pour que d'autres récoltent ailleurs et plus
tard."
[Bernard Werber]
